apps/web/components/booking/pages/AvailabilityPage.tsx

```diff
@@ -63,7 +63,9 @@ import type { AvailabilityTeamPageProps } from "../../../pages/team/[slug]/[type
 dayjs.extend(utc);
 dayjs.extend(customParseFormat);
 
-type Props = AvailabilityTeamPageProps | AvailabilityPageProps | DynamicAvailabilityPageProps;
+type Props = (AvailabilityTeamPageProps | AvailabilityPageProps | DynamicAvailabilityPageProps) & {
+  currency?: string;
+};
```

Comment:
Let's add currency to each one to keep inference 

```suggestion
type Props = (AvailabilityTeamPageProps | AvailabilityPageProps | DynamicAvailabilityPageProps)
```